schema/iiif_3_0.json

```diff
@@ -308,20 +308,126 @@
                             "logo": {
                                 "type": "array",
                                 "items": { "$ref": "#/classes/resource" }
-                            },    
+                            },
                             "seeAlso": { "$ref": "#/classes/seeAlso" }
                         }
                     }
                 ]
             }
         },
+        "collection": {
+            "allOf": [
+                { "$ref": "#/types/class" },
+                {
+                    "type": "object",
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "pattern": "^Collection"
+                        },
+                        "metadata": { "$ref": "#/classes/metadata" },
+                        "summary": { "$ref": "#/types/lngString" },
+                        "requiredStatement": { "$ref": "#/types/keyValueString" },
+                        "rights": {
```

Comment:
I think we should move rights into  a class (so it can be referenced by #/classes/rights) partly because we now have two copies of this and also looking ahead to https://github.com/IIIF/presentation-validator/issues/79